libobs/obs-windows.c

```diff
@@ -64,13 +64,42 @@ char *find_libobs_data_file(const char *file)
 	struct dstr path;
 	dstr_init(&path);
 
+	WCHAR *env_path = 0;
+	DWORD env_path_sz = 
+		GetEnvironmentVariableW(L"OBS_DATA_PATH", NULL, 0);
+
+	env_path = bmalloc(env_path_sz * sizeof(WCHAR));
+
+	if (env_path_sz) {
+		GetEnvironmentVariableW(
+		    L"OBS_DATA_PATH",
```

Comment:
Indent here for these parameters feels inconsistent with the rest of the project.